app/src/app/api/v0/city/[city]/inventory/route.ts

```diff
@@ -22,6 +22,8 @@ export const POST = apiHandler(
     if (!inventory) {
       inventory = await db.models.Inventory.create({
         ...body,
+        globalWarmingPotentialType: body.globalWarmingPotential,
+        inventoryType: body.inventoryGoal,
```

Comment:
This should not be necessary. All properties of body are already being copied over in line 24 (`...body`) using the spread operator.